example/example/tests/test_views.py

```diff
@@ -39,3 +41,63 @@ def test_proxy_s3_view(api_client, mock_data_client):
     url = f"{reverse(view_name)}?{invalid_url_params}"
     response = api_client.get(url)
     assert response.status_code == status.HTTP_500_INTERNAL_SERVER_ERROR
+
+
+@pytest.mark.django_db
+class TestUserTracking:
+    def test_submitting_non_list(self, api_client):
+        """
+        Ensure data posted to view is an array/list.
+        """
+        log_data = {"content": {"key": "Value 1", }, "tags": ["dataset"]}
+
+        url = reverse("log-tracking")
+
+        response = api_client.post(url, log_data, format="json")
+
```

Comment:
So, this is a minor comment.  And it won't stop me from approving it.  But in general, b/c you're using DRF you should be returning JSON for your Responses.  In this case you are just returning a string. Better to do:

```
response = api_client.post(whatever)
content = response.json()
assert content["some_key"] = "some value"
```

Reply:
Yes, for a POST to say create a user, I would normally return the object created, which would include it's `id`, which I've always understood as good practice. I didn't do this as I didn't see the response as particularly useful, basically it would just be returning the exact same data, but with the primary key of each object and `created` date. Frankly, I'm happy to do this, it doesn't hurt and there is no onus on the client to do anything with the response, but it is there if it wants to.

I guess my problem is in the how to do this, it's not as if I have access to the DB data, or maybe I do and I'm not smart enough to understand how. How would you get the data logged, for this situation?

Reply:
Well there are two ways to do this.  If you want to return the actual created logs, then you would have to find them in the database and return them using the existing serializer:

```
created_logs = DatabaseLogRecord.objects.filter(whatever)
response_content = DatabaseLogRecordSerializer(created_logs, many=True).data
```

But this may be overkill.  All I was suggesting was something like this:

```
response_content = { "detail": "whatever"}
return Response(response_content, status=status.HTTP_201_CREATED)
```
```

Reply:
Yeah, the former is what I was thinking, but couldn't think of a good way to filter the queryset as what is POST'd, could easily match other existing logs. I originally thought that if I created the `created` date manually, that combined with say `orgId` would work, but that would mean `orgId` being mandatory, which we don't really want.

I will make it more like the latter suggestion.